Resources/Prototypes/Catalog/VendingMachines/Inventories/secdrobe.yml

```diff
@@ -17,8 +17,11 @@
     ClothingUniformJumpsuitSecBlue: 3
     ClothingUniformJumpskirtSecBlue: 3 # DeltaV - new skirt sprited, added to secdrobe
     ClothingHeadsetSecurity: 3
-    ClothingOuterWinterSec: 2
-    ClothingOuterCoatGreatcoat: 2 # DeltaV - added greatcoats to SecDrobe. Surplus, reminds the officers of the good times.
+    ClothingHeadsetPrison: 3 # DeltaV - prison headsets in secdrobe
+    ClothingOuterCoatStasec: 2 # DeltaV - replace ClothingOuterStasecSweater # DeltaV - add sweaters to secdrobe
+    ClothingOuterStasecStasec: 2 # DeltaV - replace ClothingOuterWinterSec
+    ClothingOuterCoatStasec: 2 # DeltaV - replace ClothingOuterWinterSec
+    ClothingOuterGreatcoatStasec: 2 # DeltaV - added greatcoats to SecDrobe. Surplus, reminds the officers of the good times.
```

Comment:
All of this needs to get cleaned up. I suggest using the following:
```diff
...
    ClothingHeadsetPrison: 3 # DeltaV - prison headsets in secdrobe
+    ClothingOuterStasecSweater: 2 # DeltaV - add sweaters to secdrobe
+    ClothingOuterCoatStasec: 2 # DeltaV - replace ClothingOuterWinterSec
+    ClothingOuterGreatcoatStasec: 2 # DeltaV - added greatcoats to SecDrobe. Surplus, reminds the officers of the good times.
##    ClothingOuterArmorBasic: 2 # DeltaV - moved body armour from SecDrobe to SecTech
...
 ```

Reply:
Yeah, I was a bit worn out from jumping EE 10 engine versions pffft. Thanks!